content/docs/user-guide/how-to/share-a-dvc-cache.md

```diff
@@ -77,44 +85,3 @@ If you're using Git, commit the changes to your project's config file (usually
 $ git add .dvc/config
 $ git commit -m "config external/shared DVC cache"
 ```
-
-## Examples
```

Comment:
why removing this?

Reply:
This isn't removed. Git thinks I moved the old use case into this new file. Explanation in https://github.com/iterative/dvc.org/pull/2482#pullrequestreview-660479451

Reply:
I could try to undo everything, change the other file first in a commit, and then add this file in a separate commit so the diff is cleaner.